packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -381,6 +381,10 @@ class Frame extends React.Component {
   handleKeyDown(event) {
     const { keyCode, target } = event;
 
+    if (this.state.isOpen) {
+      event.stopPropagation();
```

Comment:
do we always want to stop propagation if the select is open or only when the escape key is closed?

Reply:
Done cac04d4